### DISCUSSION TITLE
Listing all URLs?

### DISCUSSION THREAD

**Discussion by shahzaib** [ash-jc-allen/short-url#141] [Q&A] [answered]
Hey is there a way to list all the shorted URls?

**Comment by ash-jc-allen** [comment, marked as answer]
Hey!

The `ShortURL` class is a normal Laravel model, so you could get all of your Short URLs by using something like:

```php
ShortURL::all();
```

Hopefully this helps a bit? 🙂

**Reply by shahzaib** [reply]
@ash-jc-allen ah thanks mate, you're a star! ⭐
